cloudinit/config/cc_ansible.py

```diff
@@ -133,23 +146,30 @@ def is_installed(self) -> bool:
         return bool(which("ansible"))
 
 
-def handle(
-    name: str, cfg: Config, cloud: Cloud, log: Logger, args: list
-) -> None:
+def handle(name: str, cfg: Config, cloud: Cloud, _, __) -> None:
```

Comment:
For consistency, let's not change this function signature.

Reply:
I typically avoid churn like this, but in this case I was trying to dispose of the unused `Logger` annotation. Either way, fine by me. +1